packages/manager/apps/zimbra/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@ovh-ux/manager-zimbra-app",
+  "version": "0.0.0",
+  "description": "Zimbra Project",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/apps/zimbra"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "scripts": {
+    "build": "tsc && vite build",
+    "dev": "tsc && vite",
+    "start": "lerna exec --stream --scope='@ovh-ux/manager-zimbra-app' --include-dependencies -- npm run build --if-present",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-zimbra-app' --include-dependencies -- npm run dev --if-present",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-zimbra-app' --include-dependencies -- npm run dev:watch --if-present",
+    "test:e2e": "tsc && npx playwright test --headed",
+    "test:e2e:ci": "tsc && npx playwright test",
+    "test:e2e:script": "tsc && node ../../../../scripts/run-playwright.js"
+  },
+   "dependencies": {
+    "@ovh-ux/manager-config": "^7.2.0",
+    "@ovh-ux/manager-core-api": "*",
+    "@ovh-ux/manager-react-core-application": "*",
+    "@ovh-ux/manager-core-utils": "*",
+    "@ovh-ux/shell": "^3.0.0",
+    "@tanstack/react-query": "^5.8.3",
+    "@tanstack/react-query-devtools": "^5.8.3",
+    "@ovhcloud/ods-common-theming": "^16.6.0",
```

Comment:
we need to update ods version 17.1.0

Reply:
It's applied. I changed the package.json and the imports